diff --git a/node_modules/@sanity/next-loader/dist/index.js b/node_modules/@sanity/next-loader/dist/index.js
index d7cdc44..40ee646 100644
--- a/node_modules/@sanity/next-loader/dist/index.js
+++ b/node_modules/@sanity/next-loader/dist/index.js
@@ -29,7 +29,7 @@ function defineLive(config) {
     const stega = _stega ?? (stegaEnabled && studioUrlDefined && (await draftMode()).isEnabled), perspective = _perspective ?? ((await draftMode()).isEnabled ? (await cookies()).has(perspectiveCookieName) ? sanitizePerspective(
       (await cookies()).get(perspectiveCookieName)?.value,
       "previewDrafts"
-    ) : "previewDrafts" : "published"), useCdn = perspective === "published", revalidate = fetchOptions?.revalidate ?? process.env.NODE_ENV === "production" ? !1 : void 0, { syncTags } = await client.fetch(query, await params, {
+    ) : "previewDrafts" : "published"), useCdn = perspective === "published", revalidate = fetchOptions?.revalidate ?? (process.env.NODE_ENV === "production" ? !1 : void 0), { syncTags } = await client.fetch(query, await params, {
       filterResponse: !1,
       perspective,
       stega: !1,
diff --git a/node_modules/@sanity/next-loader/src/defineLive.tsx b/node_modules/@sanity/next-loader/src/defineLive.tsx
index 7169d54..713060d 100644
--- a/node_modules/@sanity/next-loader/src/defineLive.tsx
+++ b/node_modules/@sanity/next-loader/src/defineLive.tsx
@@ -202,7 +202,7 @@ export function defineLive(config: DefineSanityLiveOptions): {
         : 'published')
     const useCdn = perspective === 'published'
     const revalidate =
-      (fetchOptions?.revalidate ?? process.env.NODE_ENV === 'production') ? false : undefined
+      fetchOptions?.revalidate ?? (process.env.NODE_ENV === 'production' ? false : undefined)
 
     // fetch the tags first, with revalidate to 1s to ensure we get the latest tags, eventually
     const {syncTags} = await client.fetch(query, await params, {
